examples/operator-rbac-openshift-tls/1-setup-operator-rbac.ipynb

```diff
@@ -0,0 +1,758 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Feast Operator with RBAC Configuration\n",
+    "## Objective\n",
+    "\n",
+    "This demo provides a reference implementation of a runbook on how to enable Role-Based Access Control (RBAC) for Feast using the Feast Operator with the Kubernetes authentication type. This serves as useful reference material for a cluster admin / MLOps engineer.\n",
+    "\n",
+    "The demo steps include deploying the Feast Operator, creating Feast instances with server components (registry, offline store, online store), and Feast client testing locally and fom the Kubernetes. The goal is to ensure secure access control for Feast instances deployed by the Feast Operator.\n",
+    " \n",
+    "Please read these reference documents for understanding the Feast RBAC framework.\n",
+    "- [RBAC Architecture](https://docs.feast.dev/v/master/getting-started/architecture/rbac) \n",
+    "- [RBAC Permission](https://docs.feast.dev/v/master/getting-started/concepts/permission).\n",
+    "- [RBAC Authorization Manager](https://docs.feast.dev/v/master/getting-started/components/authz_manager)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deployment Architecture\n",
+    "In this notebook, we will deploy a distributed topology of Feast services, which includes:\n",
+    "\n",
+    "* `Registry Server`: Handles metadata storage for feature definitions.\n",
+    "* `Online Store Server`: Uses the `Registry Server` to query metadata and is responsible for low-latency serving of features.\n",
+    "* `Offline Store Server`: Uses the `Registry Server` to query metadata and provides access to batch data for historical feature retrieval.\n",
+    "\n",
+    "Additionally, we will cover:\n",
+    "* RBAC Configuration with Kubernetes Authentication for Feast resources."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "* Kubernetes Cluster\n",
+    "* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) Kubernetes CLI tool."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install Prerequisites\n",
+    "The following commands install and configure all the prerequisites on a MacOS environment. You can find the\n",
+    "equivalent instructions on the offical documentation pages:\n",
+    "* Install the `kubectl` cli.\n",
+    "* Install Kubernetes and Container runtime (e.g. [Colima](https://github.com/abiosoft/colima)).\n",
+    "  * Alternatively, authenticate to an existing Kubernetes or OpenShift cluster.\n",
+    "  \n",
+    "```bash\n",
+    "brew install colima kubectl\n",
+    "colima start -r containerd -k -m 3 -d 100 -c 2 --cpu-type max -a x86_64\n",
+    "colima list\n",
```

Comment:
does this belong in the openshift example?

Reply:
Nop, just copied from `rbac` example as is. Removed